hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -588,4 +591,38 @@ protected boolean isAppendBlobEnabled() {
   protected AbfsServiceType getAbfsServiceType() {
     return abfsConfig.getFsConfiguredServiceType();
   }
+
+  /**
+   * Create directory with implicit parent directory.
+   * @param path path to create. Can be relative or absolute.
+   */
+  protected void createAzCopyDirectory(Path path) throws Exception {
+    String sasToken = getRawConfiguration().get(FS_AZURE_TEST_FIXED_SAS_TOKEN);
```

Comment:
Should have check that this config is present to proceed.

Reply:
Taken